edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.adoc

```diff
@@ -10,19 +10,20 @@ The {lcao} needs all your OADP resources, extra manifests, and custom catalog so
 
 include::modules/cnf-image-based-upgrade-prep-oadp.adoc[leveloffset=+1]
 
-include::modules/cnf-image-based-upgrade-prep-extramanifests.adoc[leveloffset=+1]
-
-include::modules/cnf-image-based-upgrade-prep-catalogsource.adoc[leveloffset=+1]
-
-////
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image.adoc#cnf-image-based-upgrade-shared-varlibcontainers[Configuring a shared container directory for the image-based upgrade]
+* xref:../../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image.adoc#cnf-image-based-upgrade-shared-container-directory_shared-container-directory[Configuring a shared container directory between ostree stateroots]
 
 * xref:../../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc[About installing OADP]
 
-* xref:../../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
+include::modules/cnf-image-based-upgrade-prep-extramanifests.adoc[leveloffset=+1]
+
+include::modules/cnf-image-based-upgrade-prep-catalogsource.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
 
 * xref:../../../operators/understanding/olm/olm-understanding-olm.adoc#olm-catalogsource_olm-understanding-olm[Catalog source]
-////
\ No newline at end of file
+
+* xref:../../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
```

Comment:
Links to "Performing an image-based upgrade for single-node OpenShift clusters" if you want to fix the friend reference text post GA